recipe/build.sh

```diff
@@ -2,11 +2,6 @@
 
 export CFLAGS="-I$PREFIX/include"
 export LDFLAGS="-L$PREFIX/lib"
-
-libtoolize
-aclocal
-autoreconf -i
```

Comment:
These are not needed if the tarballs from freedesktop.org as they contain the prerendered `configuration` and related files.  The original recipe used a archive GitHub mirror of the source code of the package which did not include these files.